lib/libutee/include/scp03_pta.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2020, Foundries Limited
+ */
+
+#ifndef __SCP03_PTA_CLIENT_H
+#define __SCP03_PTA_CLIENT_H
+
+#define PTA_SCP03_UUID { 0xbe0e5821, 0xe718, 0x4f77, \
+			{ 0xab, 0x3e, 0x8e, 0x6c, 0x73, 0xa9, 0xc7, 0x35 } }
+
+/*
+ * Enable SCP03 support on the SE
+ *
+ * in	params[0].a = (0) do not rotate keys,  (1) rotate keys
```

Comment:
For consistency for other PTA API header files, could you change to:
` * [in]  value[0].a	0 to not rotate keys, 1 to rotate keys`

By the way, this key rotation seems very SE050 specific. I think this should be a SE050 specific command, not a SCP03 generic command.

Reply:
The thing is that rotation can't  be done until SCP03 is enabled first as it would expose the new keys.
And since the platform already knows when SCP03 is enabled, hence why I chose to enable and optionally rotate in a single command. I think it makes sense

Reply:
> For consistency for other PTA API header files, could you change to:
> * [in] value[0].a 0 to not rotate keys, 1 to rotate keys

Minor stuff but can you address it?

> The thing is that rotation can't be done until SCP03 is enabled first as it would expose the new keys.
And since the platform already knows when SCP03 is enabled, hence why I chose to enable and optionally rotate in a single command. I think it makes sense

Ok. This only point that puzzles me is that this is SE050 specific, not SCP03 generic.
Maybe this PTA (or only this command) should be SE050 specific, not SCP03 generic.